sympy/plotting/tests/test_plot.py

```diff
@@ -60,6 +60,13 @@ def save(self):
     def close(self):
         pass
 
+def test_basic_plotting_backend():
+    x = Symbol('x')
+    try:
+        plot(x, (x, 0, 3), backend='text')
+        plot(x**2 + 1, (x, 0, 3), backend='text')
+    except TypeError as exc:
+        assert False, f"{exc}"
```

Comment:
I don't think that you should catch errors here.
It is often pointless for test code to catch errors because the pytest or the test framework does actually handles the thrown errors. If you catch the errors, it can just obscure the error messages or stack trace.

```suggestion
    plot(x, (x, 0, 3), backend='text')
    plot(x**2 + 1, (x, 0, 3), backend='text')
```